app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -225,20 +237,28 @@ class BookmarkFragment : Fragment(), CoroutineScope, BackHandler, AccountObserve
                         it.item.copyUrl(context!!)
                         FenixSnackbar.make(view!!, FenixSnackbar.LENGTH_LONG)
                             .setText(context!!.getString(R.string.url_copied)).show()
+                        requireComponents.analytics.metrics.track(Event.CopyBookmark)
                     }
                     is BookmarkAction.Share -> {
-                        it.item.url?.let { url -> requireContext().share(url) }
+                        it.item.url?.let { url ->
```

Comment:
nit: Should we use `apply` here?

Reply:
Is this to avoid a named lambda argument? This doesn't seem a good reason, but I'll do it as it doesn't really matter for this simple case which one I use.